x/tss/keeper/grpc_query.go

```diff
@@ -58,3 +62,71 @@ func (q Querier) AssignableKey(c context.Context, req *types.AssignableKeyReques
 
 	return &types.AssignableKeyResponse{Assignable: !assigned}, nil
 }
+
+// ValidatorKey returns a map of active multisig role key ids to pub keys of a validator
+func (q Querier) ValidatorKey(c context.Context, req *types.ValidatorKeyRequest) (*types.ValidatorKeyResponse, error) {
+	ctx := sdk.UnwrapSDKContext(c)
+
+	valAddress, err := sdk.ValAddressFromBech32(req.Address)
+	if err != nil {
+		return nil, err
+	}
+
+	validator := q.staker.Validator(ctx, valAddress)
+	if validator == nil {
+		return nil, fmt.Errorf("not a validator")
+	}
```

Comment:
is this necessary? It would later break anyway, wouldn't it?

Reply:
The query will silently return an empty map in this case which is not very clear. Might be worth explicitly checking this on vald startup too in a future PR.